<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Yash Agency &amp; Chaitanya Enterprise</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>yash logo.jpeg</normaloff>
    <normalon>yash logo.jpeg</normalon>yash logo.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>1261</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News701 BT</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>YASH AGENCY &amp; CHAITANYA ENTERPRISE</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>1800</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>News706 BT</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>20.png</normaloff>20.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Y">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>yash logo.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>95</y>
      <width>2471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="stockButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>270</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Stocks</string>
    </property>
   </widget>
   <widget class="QWidget" name="QuickInfoScreen" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>450</width>
      <height>160</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Quick Info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>60</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Today</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Gross Sale</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount Received</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>151</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount Due</string>
     </property>
    </widget>
    <widget class="QLabel" name="grossSale_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>68</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Rs.</string>
     </property>
    </widget>
    <widget class="QLabel" name="amountReceived_label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>68</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Rs.</string>
     </property>
    </widget>
    <widget class="QLabel" name="amountDue_label">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>68</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Rs. </string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>104</y>
      <width>20</width>
      <height>980</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="salesButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SALES</string>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>900</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>800</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>730</y>
      <width>300</width>
      <height>225</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="lowStockButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777213</height>
      </size>
     </property>
     <property name="text">
      <string>Low Stock Items :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unpaidInvoiceButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Unpaid Invoices :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>  VITAL STATS</string>
     </property>
    </widget>
    <widget class="QLabel" name="lowStock_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>40</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="unpaidInvoice_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>40</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unpaidPurchaseButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Unpaid Purchases:</string>
     </property>
    </widget>
    <widget class="QLabel" name="unpaidPurchases_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>40</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="purchaseButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>200</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>PURCHASE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="invoiceButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>340</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Invoices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quotationButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>410</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Quotations</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>730</y>
      <width>750</width>
      <height>225</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>750</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>  QUICK SEARCH</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="stockItem_RB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Stock Item</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="invoices_RB">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Invoices</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="client_RB">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>125</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Client</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="supplier_RB">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Supplier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_Edit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>625</x>
       <y>120</y>
       <width>80</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>News701 BT</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="search_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>170</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addClientButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSupplierButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Supplier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addExpenseButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Expense</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setReminderButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>411</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Reminder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paymentInButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment IN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paymentOutButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>480</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Payment OUT</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pie_chart">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>650</y>
      <width>450</width>
      <height>300</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>610</y>
      <width>450</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>  Distribution in Last 30 Days</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="bar_graph">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>1150</width>
      <height>550</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily Expense Distribution</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
